src/main/puzzle/tutorial/skill-tally-item.gd

```diff
@@ -8,16 +9,14 @@ This is used during tutorials so the player can see what to do.
 # if 'true' the skill tally will be shown as '58%' instead of '52/90'
 export (bool) var show_as_percent: bool
 
-# path of the Puzzle node to monitor
```

Comment:
Preserve this comment, `the Puzzle to monitor`

Reply:
Preserved comment, `puzzle to monitor for things such as moving the piece and clearing lines`